packages/@ember/-internals/glimmer/tests/integration/helpers/fn-test.js

```diff
@@ -127,7 +127,7 @@ moduleFor(
           arg1: 'foo',
           arg2: 'bar',
         });
-      }, /You must pass a function as the `fn` helpers first argument, you passed null/);
+      }, /You must pass a function as the `fn` helpers first argument, you passed null from this.myFunc/);
```

Comment:
I think a slight tweak to the wording here could be "you passed `this.myFunc` to `fn` but it was null", would be more clear what the user had passed to the helper